src/main/webapp/query-control.js

```diff
@@ -20,6 +20,7 @@ function registerServiceWorker() {
 // to be reformatted and visualized.
 function passQuery() {
   document.getElementById('map-title').innerText = '';
+  document.getElementById('data-table').innerHTML = '';
   am4core.disposeAllCharts();
   document.getElementById('more-info').innerText = 'Please wait. Loading...';
   document.getElementById('result').style.display = 'block';
```

Comment:
We've got a lot of stuff going on here at this point. Maybe we should make a separate smaller function, something like 'clearPreviousResult', and put all of that in there. Then it would make sense to call clearPreviousResult each time a new query is passed in.

Reply:
Done!